doc/release/matrix.md

```diff
@@ -0,0 +1,12 @@
+# Support Matrix
+
+Please find below the recommended Linux distribution and third-party software
+that have been tested with the different tags.
+
+| **Tag**  | CentOS | OpenSuse | Ubuntu | MOFED  |
+| -------- | ------ |----------| -------|--------|
+| v1.0.1   |  7.7   |    No    |   No   |  5.0.1 |
+| v1.1.2   |  7.8   |   15.2   |  20.04 |  5.1.x |
```

Comment:
Is there any implication that RHEL is actually supported because CentOS is supported or are we specifically not including it in the matrix?  Same goes for SLES I suppose?

The thing to note that is for most regular RHEL users, there is no concept of the point release 7.7, 7.8, etc.  They just subscribe and pay for support on RHEL 7 and at the point where CentOS is doing a point release, RHEL users are just automatically upgraded[1] to the equivalent point release.  There is an exception for people who pay in addition to the regular support contract, for an EUS contract, which does allow them to pin to a point release and continue to get updates on it.  But it is optional and more money, so there will be at least some amount of people out there not subscribing to it.

[1] TBH, I don't know if RHEL users have a "point release upgrade flag day" sort of event (like CentOS users would upgrading from 7.7 to 7.8) of if the update from 7.7 to 7.8 happens gradually between the points in time that CentOS releases 7.7 to 7.8.  I.e. a "rolling release" type methodology.

Reply:
That's a good point and we need to discuss this in more details. Most of our production deployment will be using RHEL or SLES, while we are validating CentOS and OpenSuse only.
For the time being, i just would like to document the "working" data points.